src/domains/spree/getSpree.ts

```diff
@@ -11,24 +12,13 @@ type Options = {
   includeBeta: boolean;
 };
 
-const filterTemplates = (templates: Spree[], options: Options) => {
-  if (options.includeBeta) return templates;
-
-  return templates.filter((e) => !e.beta);
-}
-
 /** Gets the integration from user's input. */
 const getSpree = async (options: Options): Promise<Spree> => {
   const { message } = options;
 
-  const spreeTemplates = await fetchSpreeTemplates();
-
-  const filteredTemplates = filterTemplates(spreeTemplates, options);
+  const templates = await fetchSpreeTemplates();
 
-  const choices = filteredTemplates.map((spree) => ({
-    name: spree.name,
-    value: spree
-  }));
+  const choices = buildSpreeChoices(templates, options.includeBeta, os.platform());
```

Comment:
Changing `buildSpreeChoices` arguments as discussed earlier would reduce the "os" dependency.
```suggestion
  const choices = buildSpreeChoices(templates, options);
```